lib/grandpa/models/errors.go

```diff
@@ -0,0 +1,11 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package models
+
+import "errors"
+
+var (
+	ErrBlockDoesNotExist = errors.New("block does not exist")
+	ErrVoterNotFound     = errors.New("voter is not in voter set")
```

Comment:
Actually to reduce package dependency (and avoid cycles), I created a `lib/grandpa/errors` package with `errors.go` listing those 2 errors, which can be returned from some functions both in `lib/grandpa` and in `lib/grandpa/models`. I also removed them from `lib/grandpa`, to avoid having errors defined twice (and since `lib/gandpa` imports `lib/grandpa/models`, `lib/grandpa/models` cannot import errors from its importer `lib/grandpa`).